pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -230,7 +230,7 @@ func init() {
 				},
 				{
 					APIGroups: []string{""},
-					Verbs:     sets.NewString("get", "list", "watch"),
+					Verbs:     sets.NewString("get", "list", "watch", "update"),
```

Comment:
We need the permission to enable adoption of replica sets (pods may be relabeled).